src/gatling/simulations/uk/gov/hmcts/reform/sendletter/PdfWithCopiesSimulation.scala

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.sendletter
+
+import com.typesafe.config.{Config, ConfigFactory}
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.reform.sendletter.actions.{LettersService, S2s}
+
+import scala.concurrent.duration._
+
+class PdfWithCopiesSimulation extends Simulation {
+
+  val config: Config = ConfigFactory.load()
+
+  private val duration = config.getInt("duration_in_minutes")
+  private val serviceCount = config.getInt("service_count")
+  private val maxRetry = config.getInt("max_retry")
+
+  setUp(
+    scenario("Create letters v3")
+      .exec(S2s.leaseServiceToken)
+      .during(duration.minutes)(
+        exec(
+          LettersService.createV3,
+          pause(5.seconds, 10.seconds)
```

Comment:
I think we may want to add the pause after line 25 so we pause between checks? Pretty sure the client has a wait in there for the async endpoint so that it doesn't spam it.

Reply:
pause added.